modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CustomViewsControls.js

```diff
@@ -40,12 +47,25 @@ const CustomViewsControls = () => {
 				<ClayInput
 					autoFocus={true}
 					id={`${namespace}customViewNameInput`}
+					ref={customViewNameInputRef}
 					type="text"
 				/>
 			</ClayForm.Group>
 		);
 	};
 
+	const getNextCustomViewId = () => {
+		const ids = Object.keys(customViews);
+
+		let nextId = 1;
+
+		if (ids.length) {
+			nextId = Math.max(...ids.map((item) => Number(item))) + 1;
```

Comment:
Since we're adding `+1`, why Math.max still needed?

Reply:
We plan to support deletion of custom view. So, you can end up with random stored ids. This code takes maximum of all existing IDs and adds 1. For example, if we have existing `[1, 6, 12, 7]`, it will generate 13, max + 1, that we know does not exist.

Reply:
Forget my comment, I forgot `.max` receives ...args